containers/Makefile

```diff
@@ -135,14 +128,20 @@ $(VERIFIER_SOURCE_FROM_CONCENT_API): build/verifier/%: build/concent-api/%
 	mkdir --parents "build/verifier/"
 	cp "$<" "$@"
 
-build/verifier/golem/scripts/: $(GOLEM_DIR)/apps/rendering/resources/scripts/
+build/verifier/golem/scripts/: $(GOLEM_DIR)apps/rendering/resources/scripts/
 	mkdir --parents "$@"
 	cp  --recursive "$<." "$@"
 
 
-build/verifier/golem/imgverifier-requirements.txt: $(GOLEM_DIR)/apps/rendering/resources/images/requirements.txt
+build/verifier/golem/imgverifier-requirements.txt: $(GOLEM_DIR)apps/rendering/resources/images/requirements.txt
 	cp "$<" "$@"
 
+$(CONCENT_DIR): build/package-builder/checkout-concent-version.sh /var/concent-data/concent
+	"$<" "$@"
+
+$(GOLEM_DIR): build/package-builder/checkout-golem-version.sh /var/concent-data/golem
+	"$<" "$@"
```

Comment:
Please hard-code the script name here. It's easier to understand that way.

Reply:
Done.